internal/integtest/setup.go

```diff
@@ -102,7 +102,7 @@ func HNCBeforeSuite() {
 	Expect(err).ToNot(HaveOccurred())
 
 	By("creating reconcilers")
-	err = setup.CreateReconcilers(k8sManager, forest.NewForest(), 100, true)
+	err = setup.CreateReconcilers(k8sManager, forest.NewForest(), 100, false, true)
```

Comment:
Could it make sense to create some constants to make the code more readable? What does `false, true` really mean here? `ReadOnlyMode, UseFakeClient` would read better IMO.

Reply:
Whoops, I just removed the hold. This is an easy followup.